src/aliases/api/handlers/handlers.go

```diff
@@ -85,15 +84,18 @@ func (h *AliasHandler) createAlias(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	eAlias := types.FormatAlias(regName, key, aliasReq.Value)
+	eAlias := types.FormatAlias(regName, key, aliasReq.AliasValue)
```

Comment:
function belongs in `formatters` folder.

Reply:
I don't think adding folders helps that much in comprehending the project, but I'll do it to be more cohesive with the rest of the project.